admin/tools/docker/deployment/swarm/manager/env-docker.sh

```diff
@@ -0,0 +1,27 @@
+# Configuration file sourced by Swarm scripts
+# Allow to customize Docker container execution
+
+# VERSION can be a git ticket branch but with _ instead of /
+# example: u_fjammes_DM-4295
+VERSION=tickets_DM-7139
```

Comment:
I have to use tickets_DM-7139 here, because dev container do not support swarm configuration yet and test fails.